templates/mysql.service.j2

```diff
@@ -37,3 +37,16 @@ Restart=on-failure
 RestartPreventExitStatus=1
 RuntimeDirectory=mysqld
 RuntimeDirectoryMode=755
+PrivateTmp={{ mysql_PrivateTmp | default('false') }}
```

Comment:
It would be nice to change that variable name to `mysql_private_tmp` in order to be symmetrical with the other vars (change that in default vars sample too pls) :)

Reply:
Done :)